### ISSUE TITLE
Implement (or document) the use of require in KobraScript

### ISSUE THREAD

**Issue by rtoal** [AuthorOfTheSurf/KobraScript#66]
KobraScript runs on the Node ecosystem, just like JavaScript, CoffeeScript, and many other languages. Therefore KobraScript programmers should have access to every single npm module via `require`. If this is not implemented, please implement it. If it is, place an example on the readme.

**Comment by ebramanti** [comment]
@rtoal I don't believe this is implemented yet in the language, it's in the pipeline though. Thanks for the issue report!
